tests/test_gen2to3.py

```diff
@@ -38,7 +38,7 @@
 class Gen2ConvertTestCase(lsst.utils.tests.TestCase):
 
     def setUp(self):
-        self.butler = Butler(REPO_ROOT, run="shared/ci_hsc")
+        self.butler = Butler(REPO_ROOT, collections="shared/ci_hsc")
```

Comment:
I think this line is a bug: it should be `collection`, not `collections`. Did you run jenkins with ci_hsc?

Reply:
on this ticket it is called collections. Yes, Jenkins did run fine last week.

Reply:
Although the docs say it should be `collections=["shared/ci_hsc"]` so I'm a bit worried this turned into a collection for each character in the collection name.

Reply:
I'll bet the latter happened, yes. The convert tests have explicit tests for the created collections because of this.

Reply:
The collection-expression parsing code can handle both single `str` values and iterables thereof (and a lot more).  Was there a concrete problem seen, or was this just a guess that there might be one?

Reply:
Did I miss that in the docs?

Reply:
Probably; there are enough types supported that it's hard to notice any one of them.  I'd link to them but of course they're on a ticket branch still.